renderer/components/editor/conversion/conversion-details.tsx

```diff
@@ -1,10 +1,10 @@
 import {UseConversionState} from 'hooks/editor/use-conversion';
 
 const ConversionDetails = ({conversion}: {conversion: UseConversionState}) => {
-  const message = conversion?.message ?? 'Loading…';
+  const message = conversion?.message || 'Loading…';
```

Comment:
This is because `message` is initialized with `''` in the main process.

Reply:
I would add a code comment so we don't try to revert to `??` in the future.

Reply:
But wouldn't it be better to not initialize it with `''`?

Reply:
Yeah that's fair. Mostly I just didn't want to make it nullable. But I could also initialize it with 'Loading' since that's the value it's set to in the UI when it's empty